dev/ush/make_ee2_links.sh

```diff
@@ -0,0 +1,470 @@
+#!/bin/bash
+PS4='+ $LINENO: '
+
+set -eux
+cwd=${PWD}
+cd C384C192/20240610/gdas.20230401/18/analysis/atmos
+ln -s gdas.t18z.abias gdas.t18z.abias.txt
+ln -s gdas.t18z.abias_air gdas.t18z.abias_air.txt
+ln -s gdas.t18z.abias_int gdas.t18z.abias_int.txt
+ln -s gdas.t18z.abias_pc gdas.t18z.abias_pc.txt
+ln -s gdas.t18z.radstat gdas.t18z.radstat.tar
+cd "${cwd}"
+
+cd C384mx025/20241120/gdas.20210701/00/analysis/atmos
+ln -s gdas.t00z.abias gdas.t00z.abias.txt
+ln -s gdas.t00z.abias_air gdas.t00z.abias_air.txt
+ln -s gdas.t00z.abias_int gdas.t00z.abias_int.txt
+ln -s gdas.t00z.abias_pc gdas.t00z.abias_pc.txt
+ln -s gdas.t00z.radstat gdas.t00z.radstat.tar
+cd "${cwd}"
+
+cd C48C48mx500/20250327/enkfgdas.20210323/12
+cwd_4848500=${PWD}
+for dir in mem*; do
+  cd "${dir}/analysis/ocean"
+  ln -s gdas.t12z.ocninc.nc gdas.t12z.mom6_increment.i006.nc
+  cd "${cwd_4848500}"
+done
+unset cwd_4848500
+cd "${cwd}"
+
+cd C48C48mx500/20250327/enkfgdas.20210324/12
+cwd_4848500=${PWD}
+for dir in mem*; do
+  cd "${dir}/analysis/atmos"
+  ln -s enkfgdas.t12z.ratmi003.nc enkfgdas.t12z.recentered_increment.atm.i003.nc
+  ln -s enkfgdas.t12z.ratmi009.nc enkfgdas.t12z.recentered_increment.atm.i009.nc
```

Comment:
Not asking you to change it because it's inconsequential, but it would be more intuitive to have the order here (and in other places)
- i003
- inc
- i009